app/build.gradle

```diff
@@ -103,12 +103,21 @@ dependencies {
     implementation 'com.github.tfcporciuncula:flow-preferences:1.1.0'
 
     // gRPC
-    implementation "io.grpc:grpc-netty:${grpcVersion}"
+    implementation("io.grpc:grpc-netty:${grpcVersion}") {
+        exclude module: 'netty-handler'
+    }
```

Comment:
Why do we need `exclude module:` here and under `implementation('com.github.netty.netty:netty-handler:79ef0c4706')`?

I'd guess we're trying to use the `netty-handler` module from `com.github.netty.netty:netty-handler` but we want all the other `netty-*` modules to be provided by `io.grpc:grpc-netty`. If that's the case, why do we want that? Is this temporary until `io.grpc:grpc-netty` gets a new version of Netty with the PR we contributed?

Reply:
It's temporary, just a way to be able to use the jitpack version of `netty-handler`, without getting everything through jitpack.

Reply:
So will those `exclude module:` statements go away once the Netty team release a new version? If so, can you please add it to the TODO on the README since we don't know when that'll happen.

Reply:
Yup, all will go away, and already added those TODO straight on the code, on the testing PR.